src/lib/draw/InterventionLayer.svelte

```diff
@@ -28,7 +28,7 @@
     // Add points for the ends of every LineString
     let endpoints = [];
     for (let f of copy.features) {
-      if (f.geometry.type == "LineString") {
+      if (f.geometry.type == "LineString" && !f.properties.hide_while_editing) {
```

Comment:
Found an existing bug! We draw little circles at the end of a route. But when you edit a route, those circles incorrectly stick around. It's exposed by this PR, because we abuse `hide_while_editing` to hide a bunch of stuff at once